core/backend/src/IModelSchemaLoader.ts

```diff
@@ -4,10 +4,34 @@
 *--------------------------------------------------------------------------------------------*/
 
```

Comment:
@calebmshafer @ColinKerr Here's what I came up with.  Note that I'm using 'import type' for Schema to so allow the getSchema call signatures to remain unchanged (type T would result in 'unknown' causing compilation errors in the calling code like in the IModelSchemaLoader.test.ts and IModelTransformer.test.ts).  I also updated the package.json to use peerDependenciesMeta.